src/controllers/itemDetails/index.js

```diff
@@ -1361,7 +1361,7 @@ define(['loading', 'appRouter', 'layoutManager', 'connectionManager', 'userSetti
                         imageSize: 'large',
                         enableSideMediaInfo: false,
                         highlight: false,
-                        action: layoutManager.tv ? 'resume' : 'none',
+                        action: layoutManager.tv ? 'link' : 'none',
```

Comment:
Even better: you can go to item details on mobile by clicking on a row.
```suggestion
                        action: !layoutManager.desktop ? 'link' : 'none',
```